### PR TITLE
Documentation for ports and other networking things

### PR DESCRIPTION
**Pull request by cbednarski**

### CODE REVIEW

File: website/source/docs/drivers/docker.html.md [hunk 1]
```diff
@@ -16,116 +16,192 @@ and cleaning up after containers.
...
-* `privileged` - (Optional) Privileged mode gives the container full access to
-  the host. Valid options are `"true"` and `"false"` (defaults to `"false"`).
-  Tasks with `privileged` set can only run on Nomad Agents with
-  `docker.privileged.enabled = "true"`.
-
-* `dns_servers` - (Optional) A list of DNS servers for the container to use
-  (e.g. ["8.8.8.8", "8.8.4.4"]). *Docker API v1.10 and above only*
+* `hostname` - (Optional) The hostname to assign to the container. When
+  launching more than one of a task (using `count`) with this option set, every
+  container the task starts will have the same hostname.* `dns_servers` -
```
Comment (dadgar) [inline website/source/docs/drivers/docker.html.md]: `dns_servers` should be on a new line

File: website/source/docs/jobspec/networking.html.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+outbound connection.
+
+## IP Address
+
+Hosts in Nomad may have multiple network interfaces attached to them. This
+allows you to have a higher density of services, or bind to interfaces on
+different subnets (for example public vs. private subnets).
+
+Each task will receive port allocations on a single interface. The IP is passed
+to your job via the `NOMAD_IP.` environment variable.
```
Comment (dadgar) [inline website/source/docs/jobspec/networking.html.md]: Remove the period on NOMAD_IP

File: website/source/docs/jobspec/restarts.html.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+layout: "docs"
+page_title: "Nomad Restart Policies"
+sidebar_current: "docs-jobspec-restarts"
+description: |-
+  Learn how to restart tasks running on Nomad.
+---
+
```
Comment (dadgar) [inline website/source/docs/jobspec/restarts.html.md]: Remove this one?

File: website/source/docs/drivers/docker.html.md (hunk 1 above)
Comment (cbednarski) [inline website/source/docs/drivers/docker.html.md]: Oops thanks!

**Pull request merged**